haskell/nixpkgs.bzl

```diff
@@ -173,7 +176,8 @@ def haskell_register_ghc_nixpkgs(
         locale = None,
         repositories = {},
         repository = None,
-        nix_file_content = None):
+        nix_file_content = None,
+        use_worker = False):
```

Comment:
This flag will show up in the [API docs for master](https://api.haskell.build/haskell/nixpkgs.html#haskell_register_ghc_nixpkgs). Best document that this is not ready for use by regular users, yet.